### ISSUE TITLE
Pulsar standalone fails to start with java 9

### ISSUE THREAD

**Issue by sijie** [apache/pulsar#1419]
#### Expected behavior

`./bin/pulsar standalone` should be able to run with java 9

#### Actual behavior

standalone fails to start with following errors.

```
Exception in thread "main" java.lang.RuntimeException: java.lang.reflect.InvocationTargetException
	at com.ea.agentloader.AgentLoader$1.loadAgent(AgentLoader.java:244)
	at com.ea.agentloader.AgentLoader.loadAgent(AgentLoader.java:77)
	at com.ea.agentloader.AgentLoader.loadAgentClass(AgentLoader.java:148)
	at com.ea.agentloader.AgentLoader.loadAgentClass(AgentLoader.java:102)
	at org.apache.pulsar.PulsarStandaloneStarter.start(PulsarStandaloneStarter.java:178)
	at org.apache.pulsar.PulsarStandaloneStarter.main(PulsarStandaloneStarter.java:252)
Caused by: java.lang.reflect.InvocationTargetException
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
	at com.ea.agentloader.AgentLoader$1.loadAgent(AgentLoader.java:240)
	... 5 more
Caused by: java.lang.RuntimeException: java.io.IOException: Can not attach to current VM
	at com.ea.agentloader.AgentLoaderHotSpot.getVirtualMachine(AgentLoaderHotSpot.java:90)
	at com.ea.agentloader.AgentLoaderHotSpot.loadAgent(AgentLoaderHotSpot.java:56)
	... 10 more
Caused by: java.io.IOException: Can not attach to current VM
	at jdk.attach/sun.tools.attach.HotSpotVirtualMachine.<init>(HotSpotVirtualMachine.java:75)
	at jdk.attach/sun.tools.attach.VirtualMachineImpl.<init>(VirtualMachineImpl.java:58)
	at jdk.attach/sun.tools.attach.AttachProviderImpl.attachVirtualMachine(AttachProviderImpl.java:58)
	at jdk.attach/com.sun.tools.attach.VirtualMachine.attach(VirtualMachine.java:207)
	at com.ea.agentloader.AgentLoaderHotSpot.getVirtualMachine(AgentLoaderHotSpot.java:86)
	... 11 more
```

#### Steps to reproduce

use java 9

```
$ java --version
java 9.0.1
Java(TM) SE Runtime Environment (build 9.0.1+11)
Java HotSpot(TM) 64-Bit Server VM (build 9.0.1+11, mixed mode)
```

run `bin/pulsar standalone`

#### System configuration
**Pulsar version**: master
**Java**: java 9
**OS**: mac

**Comment by maskit** [comment]
Maybe this? https://github.com/electronicarts/ea-agent-loader/issues/10

**Comment by sijie** [comment]
@maskit yes. that is the issue.

**Comment by yangl** [comment]
With the introduction of JDK9, dynamically self attaching Java Agents is considered bad practice. As such we are deprecating this project and will no longer continue to maintain it. The project remains here for reference and for JDK8 users.

https://github.com/electronicarts/ea-agent-loader#project-deprecated

**Comment by merlimat** [comment]
This was fixed in #3006
